pkg/apis/flowcontrol/types.go

```diff
@@ -232,6 +237,17 @@ type ResourcePolicyRule struct {
 	// +listType=set
 	// Required.
 	Resources []string
+	// `namespaces` is a list of target namespaces that restricts
+	// matches.  A request that does not specify a target namespace
+	// (which happens both when the resource is not namespaced and
+	// when the resource is namespaced and the request is for all
+	// namespaces) matches only if this list includes "Cluster Scope".
+	// A request that specifies a target namespace matches only if
+	// either (a) this list contains that target namespace or (b) this
+	// list contains "*".
```

Comment:
I'd add this or language which covers these points: `A list containing "*" may also contain "Cluster Scope", but no other entries. A list containing "Cluster Scope" may be combined with other entries (specific namespace references) as desired. `

And also we should end with a note about: `It is the user's responsibility to make sure that the choice here matches the scope of the resource(s) selected; specifically, if the resource(s) are cluster scoped, then this list must contain "Cluster Scope" or it will fail to match any requests. At least in this alpha, the server-performed validation can't detect and protect you from this misconfiguration.`